x-pack/qa/repository-old-versions/src/test/resources/org/elasticsearch/oldrepos/custom.json

```diff
@@ -1,4 +1,7 @@
 "_default_": {
+  "_field_names": {
+    "enabled": false
+  },
```

Comment:
Don't we end up testing only with field names disabled now?

Reply:
No, this is just one of the test mappings, the others don't have this disabled.

Reply:
Also the other tests (i.e OldMappingsIT, DocValues...) are not using that parameter.